presto-main/src/test/java/com/facebook/presto/cost/TestCostCalculator.java

```diff
@@ -251,17 +236,13 @@ private void assertUnknownCostForUnknownStats(PlanNode planNode)
                 new FixedLookup(id -> UNKNOWN_COST, id -> UNKNOWN_STATS),
                 session,
                 ImmutableMap.of()))
-                .cpuUnknown()
-                .memoryUnknown()
-                .networkUnknown();
+                .hasUnknownComponents();
         new CostAssertionBuilder(costCalculatorWithEstimatedExchanges.calculateCumulativeCost(
                 planNode,
                 new FixedLookup(id -> UNKNOWN_COST, id -> UNKNOWN_STATS),
                 session,
                 ImmutableMap.of()))
-                .cpuUnknown()
-                .memoryUnknown()
-                .networkUnknown();
+                .hasUnknownComponents();
```

Comment:
Previously this method tested that `cost.isUnknown()` is true. `isUnknown()` method was renamed to `hasUnknownComponents`, therefore this assertion has semantics as it was before. I will rename the `assertUnknownCostForUnknownStats` method though.